packages/@vuepress/core/lib/client/util.js

```diff
@@ -4,6 +4,11 @@ import pageComponents from '@internal/page-components'
 
 const asyncComponents = Object.assign({}, layoutComponents, pageComponents)
 
+// TODO: reuse this function in shared-utils
+function pascalize (source = '') {
```

Comment:
Consider use a `cached` fn like https://github.com/vuejs/vue/blob/731e4d0ebc997a6c2b1fc500b08924ff7ba091b6/src/shared/util.js#L153-L159

Reply:
Considering that this function might come from shared-utils in the future, this sort of optimization will be implemented after then.